src/util.rs

```diff
@@ -130,7 +423,138 @@ pub(crate) mod testutil {
 
 #[cfg(test)]
 mod tests {
+    use core::mem::MaybeUninit;
+
     use super::*;
+    use crate::{util::testutil::AU64, FromBytes};
+
+    #[test]
+    fn test_ptr_try_cast_into_soundness() {
```

Comment:
Discussed offline; we're going to just disable symbolic alignment checking because the alternative introduces more complexity and CI test time than it's worth for a marginal benefit.